.cirrus.yml

```diff
@@ -11,4 +11,9 @@ task:
     folder: $CARGO_HOME/registry
   build_script: env PATH="$HOME/.cargo/bin:$PATH" cargo build
   test_script: env PATH="$HOME/.cargo/bin:$PATH" cargo test
+  coverage_script: |
+    pkg install -y kcov bash
+    for file in target/debug/jail-*[^\.d]; do mkdir -p "target/cov/$(basename $file)"; kcov --exclude-pattern=/.cargo,/usr/lib --verify "target/cov/$(basename $file)" "$file"; done &&
```

Comment:
`/.cargo` should be `$HOME/.cargo`.  Also, you should probably exclude the `benches` and `examples` directories (though you'll probably be ok since your glob pattern will exclude them).

Reply:
Thanks! I'm currently still figuring out how to reformulate the loop in `tcsh`...

Reply:
```tcsh
foreach file (target/debug/jail-*[^\.d])
  mkdir
  do other stuff
end
```

Reply:
Hmm...:
```
foreach file ( target/debug/jail-*[^\.d] )
/tmp/scripts3a1b1d49d4955c8486216325253fec73.sh: 4: Syntax error: "(" unexpected
```
Is this really tcsh?

Reply:
Why not `file --exec`?

Reply:
@gronke Because we basically need to exec two commands per file - see:
https://github.com/fubarnetes/libjail-rs/blob/8a6b00d846d1da9e353f5e8b495732be5c21cbcc/.cirrus.yml#L16-L19

However, now I don't quite understand the error message in https://github.com/fubarnetes/libjail-rs/runs/49792035